docs-mslearn/framework/understand/ingestion.md

```diff
@@ -108,6 +108,23 @@ At this point, you have a data pipeline and are ingesting data into a central da
 
 <br>
 
+## KPIs and metrics
+
+To ensure effective data ingestion in Azure, these KPIs provide a framework for evaluating key aspects such as completeness and quality.
+
+| **KPI** | **Definition** | **Formula** |
+|----------|-----------|-----|
+| Data completeness | Measures the extent to which all required data fields are present in the dataset and tracks the overall data completeness trend over a specified period.| Percentage of data fields that are complete and the overall data completeness over time. |
+| Data quality | Measures the percentage of successful data quality checks and the total number of data quality checks conducted within a specified period. | Number of data quality checks conducted and the percentage of successful data quality checks . |
+|Data ingestion frequency | Measures how often data is ingested into the system. | Number of data ingestion events per unit of time (daily, weekly, monthly, quarterly, annually). |
+| Data size | Measures the total volume of data ingested into the repository. | Total volume of data ingested into the repository.  |
+| Growth rate | Measure the rate at which the volume of data ingested is increasing over time. | Percentage increase of total data volume in repository per unit of time. |
+| Ingestion latency | Measures the average time taken for data to be ingested into the repository. | Average time of data ingestion latency per dataset. |
```

Comment:
Some (or maybe all) of these seem like they are per dataset or data source. Should we call that out in any way? Should we have 2 lists? One for overall KPIs and one per dataset/source? 🤔